docs/UsersGuide/source/CustomizingTheWorkflow/ConfigWorkflow.rst

```diff
@@ -1295,6 +1299,10 @@ Set parameters for customizing the :term:`UPP`.
 ``TESTBED_FIELDS_FN``: (Default: "")
    The file that lists grib2 fields to be extracted for testbed files. An empty string means no need to generate testbed files.
 
+``NUMX``: (Default: 1)
```

Comment:
I'm a bit confused from this description. Is ``i`` here the same as `ITASKS`? If not, can you expand a little bit more on the relationship between `ITASKS` and `NUMX`?

Reply:
@mkavulich For offline post, there is no `ITASKS`.  `ITASKS` is only used in inline post and is set in `ufs-weather-model/FV3/io/post_fv3.F90` as:

`numx = itasks`

UPP requires a value of `NUMX` greater than 1 to activate 2d decomposition, and rather than add `NUMX` to inline post, the `ITASKS` variable was used instead.

So, in the case of offline post, no, `i` is not the same as `ITASKS` (since there is no `ITASKS` variable in UPP).  For inline post, `i` is the same as `ITASKS`.

Reply:
Sorry, I'm still a bit confused: so inline and offline post have two different ways of enabling 2d-decomposition? NUMX, used for inline post only, is treated the same as ITASKS is in offline post?

Reply:
Documentation for `NUMX` has been updated to show that it is for offline post (while `ITASKS` is for inline post).